azure-iot-device/tests/common/pipeline_test_fixtures.py

```diff
@@ -0,0 +1,25 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+from . import pipeline_test
+from azure.iot.device.common.transport import pipeline_events_base
+
+
+@pytest.fixture
+def callback(mocker):
+    return mocker.Mock()
+
+
+@pytest.fixture
+def fake_error():
+    return pipeline_test.get_fake_error()
+
+
+@pytest.fixture
+def event():
```

Comment:
>event [](start = 4, length = 5)

can this be named to mock event or test event ?

Reply:
I disagree, it isn't a mock event. Nor is it really a "test" event. It just is a pipeline event.

---
In reply to: [285817618](https://github.com/Azure/azure-iot-sdk-python-preview/pull/94#discussion_r285817618) [](ancestors = 285817618)